public/js/index.js

```diff
@@ -2803,7 +2803,18 @@ function updateViewInner () {
   var lastMeta = md.meta
   md.meta = {}
   delete md.metaError
-  var adoc_options = Opal.hash2(['header_footer','attributes'],{ 'header_footer': true, 'attributes': ['icons=font@', 'showTitle=true']})
+
+  //extentions
+  //asciidoctor-plantuml
+  var plantuml = require('asciidoctor-plantuml');
+  plantuml.register(adoc.Extensions);
+
+  var adoc_options = Opal.hash2(['header_footer','attributes'],{ 'header_footer': true, 'attributes': ['icons=font@', 'showTitle=true']});
+
+  //add plantuml url
+  value = `:plantuml-server-url: http://www.plantuml.com/plantuml\n` + value;
```

Comment:
Do i undestand this correctly that it does add the plan-server-url implicitly to the header of each document? so the user does not need to add is manually on the one hand, but on the other hand, data is send to that server without notice?
I'd rather have a very explicit option. We could add the `:plantuml-server-url: http://www.plantuml.com/plantuml`in the first line when the user inserts [plantuml] or something similar, what do you think?

Reply:
Ahh,I think  `plantuml-server-url` is not necessary. cuz' plantuml.com is famous system.
So, I change implicit that.